File: Anul II/Medii de proiectare si programare/Laborator/lab05/Ubb.BikeContest.Model/src/data/RaceDto.cs
namespace Ubb.BikeContest.Model.data;

public class RaceDto
{
    private readonly string _name;
    private readonly int _engineCapacity;
    private readonly int _participant;

    public RaceDto(string name, int engineCapacity, int participant)
    {
        _name = name;
        _engineCapacity = engineCapacity;
        _participant = participant;
    }

    public string Name => _name;

    public int EngineCapacity => _engineCapacity;

    public int Participants => _participant;
}

File: Anul II/Medii de proiectare si programare/Laborator/lab06/model/src/main/java/com/ubb/model/data/RaceDTO.java
package com.ubb.model.data;

public class RaceDTO {
    private final String name;
    private final Integer engineCapacity;
    private final Integer participants;

    public RaceDTO(String name, Integer engineCapacity, Integer participants) {
        this.name = name;
        this.engineCapacity = engineCapacity;
        this.participants = participants;
    }

    public String getName() {
        return name;
    }

    public Integer getEngineCapacity() {
        return engineCapacity;
    }

    public Integer getParticipants() {
        return participants;
    }
}
